paas/api.j2

```diff
@@ -1,5 +1,10 @@
 {% extends "_base.j2" %}
 
+{% block health_check %}
+    health-check-type: http
+    health-check-http-endpoint: {{ routes[0].partition('/')[1:]|join("") }}/_status
```

Comment:
Does this depend on a fixed ordering of routes? I remember that biting us in the past...

Reply:
It does. It uses the same pattern as where it inherited it before here:
https://github.com/alphagov/digitalmarketplace-aws/blob/master/paas/_base.j2#L15

We could probably hard code this now we know what app we're operating on.

But might not need to change this at app now

alphagov/digitalmarketplace-antivirus-api#59
alphagov/digitalmarketplace-utils#550